runner/src/dml-handler/dml-handler.ts

```diff
@@ -6,15 +6,24 @@ import type IndexerConfig from '../indexer-config/indexer-config';
 import { type Tracer, trace, type Span } from '@opentelemetry/api';
 import { type QueryResult } from 'pg';
 
-type WhereClauseMulti = Record<string, (string | number | Array<string | number>)>;
-type WhereClauseSingle = Record<string, (string | number)>;
-
-export default class DmlHandler {
+export type PostgresRowValue = string | number | any;
+export type PostgresRow = Record<string, PostgresRowValue>;
+export type WhereClauseMulti = Record<string, (PostgresRowValue | Array<PostgresRowValue>)>;
+export type WhereClauseSingle = Record<string, PostgresRowValue>;
+
+export interface DmlHandlerI {
```

Comment:
```suggestion
export interface IDmlHandler {
```
I think this is more conventional